### DISCUSSION TITLE
there is an error in example_01 intputfile

### DISCUSSION THREAD

**Discussion by xiaojbing** [idaholab/moose#20510] [Q&A Getting Started] [answered]
this is a snippet of example_01 inputfile
```
[Executioner]
  type = Steady
  solve_type = 'PJFNK'
  file_base = 'out'
[]
```
I think the parameter _file_base='out'_ should put in the output block. Because when I run the inputfile,there is a warning like

```
*** Warning ***
/home/xjb/projects/moose/examples/ex01_inputfile/run_by_xiao/ex01.i:40: unused parameter 'Executioner/file_base'
```

**Comment by GiudGiud** [comment, marked as answer]
Hello

Thanks a lot for reporting this!
I ll fix it in an upcoming PR with other deprecated and unused syntaxes. 

Guillaume

**Comment by GiudGiud** [comment]
Hello

Thanks a lot for reporting this!
I ll fix it in an upcoming PR with other deprecated and unused syntaxes. 

Guillaume
